### ISSUE TITLE
Version 0.9.34 causes double acc verification

### ISSUE THREAD

**Issue by Marurban** [playgameservices/play-games-plugin-for-unity#1184]
I have updated plugin to latest version because AdMob didn't work properly. It fixed my problems but now I have 3 new issues:
1. After login into play games, it shows next window with acc selection (My login flow is same as in your examples and it worked before update normally). First it shows normal white popup from PlayGames, after that appears second window like from ICS or JB
2. Game really slows down 
3. Exit from game takes 5x more time

Don't know how it is possible because I didn't change anything.
I used before version 0.9.32.

**Comment by claywilkinson** [comment]
1. What is ICS or JB?
2. Where is it slowing down? Overall?  Can you share what you are seeing in the profiler?
3. What are you measuring/comparing when exiting? Can you share a log or profiler information of exiting to help understand why it is taking 5x longer?

Did you update the play-services aars in Assets/Plugins/Android?  What version are you using?

Can you go back to 0.9.33 and compare it?

**Comment by Marurban** [comment]
1. Ice Cream Sandwich / Jelly Bean

Solution for long exiting is in this topic: http://answers.unity3d.com/questions/855852/issue-applicationquit-takes-much-time-to-close-and.html

I reverted PlayGames to 0.9.33 and second authentication gone. If you need more data with this issue let me know.

**Comment by buihuuloc** [comment]
Hi @Marurban 

I'm still facing with this problem.

How can I remove double account verification? 

Thanks

**Comment by Marurban** [comment]
I returned to version 0.9.32 and it fixed this problem.

**Comment by buihuuloc** [comment]
:). Oh. Thanks.

**Comment by buihuuloc** [comment]
Unbelievable. I got an other issue when I downgrade to v32.
I need to get player Email. However version 0.9.32 doesn't work.
Is there any solution to upgrade to the latest version without double account verification?

**Comment by Marurban** [comment]
Did you update app or did clean install when you were on v32?

**Comment by buihuuloc** [comment]
I cleaned v33 in my project and imported v32.

**Comment by Marurban** [comment]
But when you install apk, you have installed 33, you downgraded to 32 or uninstalled 33 and then install 32?

**Comment by buihuuloc** [comment]
I have removed my app before reinstalled  with the new one

**Comment by Marurban** [comment]
I solved this problem by adding additional statement to my authentication method
```
if(authenticated) {
    PlayGamesPlatform.Instance.Authenticate(mAuthCallback, true);
} else if(mAuthOnStart) {
    PlayGamesPlatform.Instance.localUser.Authenticate(mAuthCallback);
}
```
```Authenticate(callback)``` doesn't recognize that user has already succesfully logged in with PlayGames account and asks to login again with Plus account. I think problem is with ```Authenticate``` method here [https://github.com/playgameservices/play-games-plugin-for-unity/blob/master/source/PluginDev/Assets/GooglePlayGames/Platforms/Native/NativeClient.cs](url)

**Comment by claywilkinson** [comment]
This is fixed in version 0.9.37 of the plugin.
